python-frontend/src/test/java/org/sonar/python/index/ClassDescriptorTest.java

```diff
@@ -43,6 +48,7 @@ public void classDescriptor() {
     FunctionDescriptor fooMethod = ((FunctionDescriptor) classDescriptor.members().iterator().next());
     assertThat(fooMethod.name()).isEqualTo("foo");
     assertThat(fooMethod.fullyQualifiedName()).isEqualTo("package.mod.A.foo");
+    assertClassDescriptors(classDescriptor, fromProtobuf(toProtobuf(classDescriptor)));
```

Comment:
I am not sure about having this test here: it seems to test `DescriptorsToProtobuf` as opposed to `ClassDescriptor`. I think it's fine for now, but we might want to move this before merging the feature branch.

Reply:
It's true that strictly speaking we should have dedicated tests for `DescriptorsToProtobuf`. However I felt this was such as simple way to leverage all our existing descriptor tests almost for free that I didn't bother with it. Especially as this really tests what we want to achieve (what we get after a serialization and deserialization is the same as what we had in input).
Of course we can re-discuss this before we merge the feature branch but indeed in the meantime I also feel this is good for now.